swan-lake/development-tutorials/java-interoperability/ballerina-ffi.md

```diff
@@ -36,7 +36,7 @@ function doSomething(int i) returns string = @java:Method {
 } external;
 ```
 
-The `@java:Method` annotation instructs the jBallerina compiler to link to the `doSomethingInJava` static method in the `a.b.c.Foo` Java class. There are a set of annotations and other utilities available in the `ballerina/jballerina.java` module to make Java interoperability work that is covered in this guide.
+The `@java:Method` annotation instructs the jBallerina compiler to link to the `doSomethingInJava` static method in the `a.b.c.Foo` Java class. Please refer [Manage platform dependencies](/learn/manage-dependencies/#manage-platform-dependencies) to add Java class dependencies to a Ballerina package. There are a set of annotations and other utilities available in the `ballerina/jballerina.java` module to make Java interoperability work that is covered in this guide.
```

Comment:
```suggestion
The `@java:Method` annotation instructs the jBallerina compiler to link the `doSomethingInJava` static method in the `a.b.c.Foo` Java class. Please refer [Manage platform dependencies](/learn/manage-dependencies/#manage-platform-dependencies) to add Java class dependencies to a Ballerina package. There are a set of annotations and other utilities available in the `ballerina/jballerina.java` module to make Java interoperability work that is covered in this guide.
```
Is it wrong to say it this way?

Reply:
I think we need the "to" here. 

We can maybe add 

"to link this function to the"

Reply:
Yeah that sounds alright as well.